aws/table_aws_elastic_beanstalk_environment.go

```diff
@@ -0,0 +1,299 @@
+package aws
+
+import (
+	"context"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/elasticbeanstalk"
+	"github.com/turbot/steampipe-plugin-sdk/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/plugin/transform"
+)
+
+//// TABLE DEFINITION
+
+func tableAwsElasticBeanstalkEnvironment(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "aws_elastic_beanstalk_environment",
+		Description: "AWS ElasticBeanstalk Environment",
+		Get: &plugin.GetConfig{
+			KeyColumns:        plugin.SingleColumn("environment_name"),
+			ShouldIgnoreError: isNotFoundError([]string{"ResourceNotFoundException"}),
+			Hydrate:           getAwsElasticBeanstalkEnvironment,
+		},
+		List: &plugin.ListConfig{
+			Hydrate: listAwsElasticBeanstalkEnvironments,
+		},
+		GetMatrixItem: BuildRegionList,
+		Columns: awsRegionalColumns([]*plugin.Column{
+			{
+				Name:        "environment_name",
+				Description: "The name of this environment.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "environment_id",
+				Description: "The ID of this environment.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "date-created",
+				Description: "The creation date for this environment.",
+				Type:        proto.ColumnType_TIMESTAMP,
+			},
+			{
+				Name:        "date_updated",
+				Description: "The last modified date for this environment.",
+				Type:        proto.ColumnType_TIMESTAMP,
+			},
+			{
+				Name:        "cname",
+				Description: "The URL to the CNAME for this environment.",
+				Type:        proto.ColumnType_STRING,
+				Hydrate:     getAwsElasticBeanstalkEnvironment,
+			},
+			{
+				Name:        "application_name",
+				Description: "The name of the application associated with this environment.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "description",
+				Description: "Describes this environment.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "endpoint_url",
+				Description: "The URL to the LoadBalancer.",
+				Type:        proto.ColumnType_STRING,
+				Hydrate:     getAwsElasticBeanstalkEnvironment,
+			},
+			{
+				Name:        "abortable_operation_in_progress",
+				Description: "Indicates if there is an in-progress environment configuration update or application version deployment that you can cancel.",
+				Type:        proto.ColumnType_BOOL,
+			},
+			{
+				Name:        "environment_arn",
```

Comment:
```suggestion
				Name:        "arn",
```
Also make this as the 3rd column